src/System Application/App/AI/src/Copilot/CopilotAICapabilities.Page.al

```diff
@@ -30,25 +30,11 @@ page 7775 "Copilot AI Capabilities"
     {
         area(Content)
         {
-            group(AllowedDataMovementOffInfo)
+            group(NormalAOAIArea)
             {
                 ShowCaption = false;
-                Visible = ((not WithinGeo) or WithinEuropeGeo) and (not AllowDataMovement);
-                InstructionalText = 'Copilot uses the Azure OpenAI Service, which isn’t available in your region. To activate Copilot capabilities, you must allow data movement.';
-            }
-
-            group(AllowedDataMovementOnInfo)
-            {
-                ShowCaption = false;
-                Visible = ((not WithinGeo) or WithinEuropeGeo) and AllowDataMovement;
-                InstructionalText = 'Copilot uses the Azure OpenAI Service, which isn’t available in your region. To keep using Copilot capabilities, you must allow data movement.';
-            }
-
-            group(AlwaysConnected)
-            {
-                ShowCaption = false;
-                InstructionalText = 'Copilot and other generative AI capabilities use Azure OpenAI Service. Your environment connects to Azure OpenAI Service in your region.';
-                Visible = WithinGeo and (not WithinEuropeGeo);
+                Visible = WithinNormalAOAI;
```

Comment:
What is "normal"?

Consider naming this something more descriptive like `AOAIServicesInRegion` (same for the bool variable and for the AL group)